actionview/test/template/asset_tag_helper_test.rb

```diff
@@ -565,9 +565,6 @@ class PlaceholderImage
     def blank?; true; end
     def to_s; "no-image-yet.png"; end
   end
-  def test_image_tag_with_blank_placeholder
```

Comment:
Yeah, but there is no reason to delete this test, even more because it was testing a different kind of object.